test/unit/display_utils_spec.js

```diff
@@ -295,6 +295,8 @@ describe("display_utils", function () {
             expect(result).toEqual(expectation);
           }
         }
+        const now = new Date();
+        expect(PDFDateString.toDateObject(now)).toEqual(now);
```

Comment:
Is there a particular reason why we can't add this case to the existing array of cases above?

Reply:
Yep, setting `now` as a key will convert it into a string and I want to test `toDateObject` with a Date.